install.sh

```diff
@@ -133,20 +133,20 @@ if [ "$(uname)" = "Linux" ]; then
     sudo apt-get install -y python3-venv
   elif type pacman >/dev/null 2>&1 && [ -f "/etc/arch-release" ]; then
     # Arch Linux
+    # Arch provides latest python version. User will need to manually install python 3.9 if it is not present
     echo "Installing on Arch Linux."
-    echo "Python <= 3.9.9 is required. Installing python-3.9.9-1"
     case $(uname -m) in
-      x86_64)
-        sudo pacman ${PACMAN_AUTOMATED} -U --needed https://archive.archlinux.org/packages/p/python/python-3.9.9-1-x86_64.pkg.tar.zst
-        ;;
-      aarch64)
-        sudo pacman ${PACMAN_AUTOMATED} -U --needed http://tardis.tiny-vps.com/aarm/packages/p/python/python-3.9.9-1-aarch64.pkg.tar.xz
+      x86_64|aarch64)
+        if ! command -v python3.9 >/dev/null 2>&1; then
+          echo "Python <= 3.9.10 is required. Please see https://wiki.archlinux.org/title/python#Old_versions for support."
+          exit 1
+        fi
```

Comment:
We already have a search that allows 3.7-3.9 below and warns with `echo "Chia requires Python version <= 3.9.9"`.  Which isn't exactly right...  maybe that should be allowed to run and this is just a no-op here?  Or not even a case?  Maybe there should be OS-specific messages included below.

Reply:
I've relocated the messaging as per your suggestion